src/shared/themes.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Theme } from './types';
+
+const themes: Theme[] = [
+  {
+    themeName: 'dark',
+    bgColor: '#353434',
+    cardBg: 'dark',
```

Comment:
These aren't really specific to cards tho, right? It's just that with bootstrap we can define an background and border and swap which one is color A and which is color B, right?

Reply:
I just set it up like this to kinda separate everything we're using in case we start doing stuff on a different component and we want to use other variants. also not every component has the bg= and border= properties